webui/webui/components/chat.py

```diff
@@ -56,52 +58,55 @@ def chat() -> rx.Component:
 
 def action_bar() -> rx.Component:
     """The action bar to send a new message."""
-    return rx.chakra.box(
-        rx.chakra.vstack(
+    return rx.center(
+        rx.vstack(
             rx.chakra.form(
                 rx.chakra.form_control(
-                    rx.chakra.hstack(
-                        rx.chakra.input(
-                            placeholder="Type something...",
-                            id="question",
-                            _placeholder={"color": "#fffa"},
-                            _hover={"border_color": styles.accent_color},
-                            style=styles.input_style,
+                    rx.hstack(
+                        rx.radix.text_field.root(
+                            rx.radix.text_field.input(
+                                placeholder="Type something...",
+                                id="question",
+                                width="30em",
+                            ),
+                            rx.radix.text_field.slot(
```

Comment:
Looks nice visually, but I wonder if we should simplify this to just an `rx.input` as the interface is already intuitive

Reply:
I like the look of this, I think it make it look more professional